webapp/src/containers/MasternodesPage/saga.tsx

```diff
@@ -115,8 +113,20 @@ export function* handleRestartNode() {
       const data = yield call(getAddressInfo, masternodeOwner);
       if (data.ismine && !data.iswatchonly) {
         const updatedConf = yield call(getConfigurationDetails);
-        updatedConf.masternode_operator = masternodeOperator;
-        updatedConf.masternode_owner = masternodeOwner;
+        updatedConf[network] = {
```

Comment:
add a check here in case updatedConf[network] is null or undefined. I think ...updatedConf[network] might break